files/zh-cn/learn/html/introduction_to_html/getting_started/index.md

```diff
@@ -674,8 +688,7 @@ textarea.onkeyup = () => {
 ```html
 <p>狗 狗 很 呆 萌。</p>
 
-<p>狗 狗        很
-         呆 萌。</p>
+<p>狗 狗 很 呆 萌。</p>
```

Comment:
Equivalent form of the previous line. We should revert this changes and add `-nolint` :)